iothub_client/tests/iothub_client_retry_control_ut/iothub_client_retry_control_ut.c

```diff
@@ -943,11 +943,11 @@ TEST_FUNCTION(Should_Retry_INTERVAL_success)
     unsigned int max_retry_time_in_secs = 19;
     RETRY_CONTROL_HANDLE handle = create_retry_control(IOTHUB_CLIENT_RETRY_INTERVAL, max_retry_time_in_secs);
 
-    int expected_retry_times[] = { 0, 5, 10, 15, 20 };
+    int expected_retry_times[] = { 0, 5, 10, 15, 20, 25, 30, 30 };
```

Comment:
Same as above: first retry should be after at least 1 second.

Reply:
Same as above.